assets/src/edit-story/utils/elementBorder.js

```diff
@@ -114,12 +105,37 @@ export function getInnerRadius(outerRadius, oneSide, otherSide) {
   return outerRadius - Math.min(outerRadius, Math.max(oneSide, otherSide)) / 2;
 }
 
-export function getBorderRadius({ borderRadius }) {
+function getPercentage(value, fullValue) {
+  if (!value || !fullValue) {
+    return 0;
+  }
+  return (value / fullValue) * 100;
+}
+
+function getCornerPercentages(borderRadius, measure) {
   if (!borderRadius) {
+    return '0%';
+  }
+  const { topLeft, topRight, bottomRight, bottomLeft } = borderRadius;
+  return `${getPercentage(topLeft, measure)}% ${getPercentage(
+    topRight,
+    measure
+  )}% ${getPercentage(bottomRight, measure)}% ${getPercentage(
+    bottomLeft,
+    measure
+  )}%`;
+}
+
+export function getBorderRadius(element) {
+  const { borderRadius, width, height } = element;
+  if (!borderRadius || !canMaskHaveBorder(element)) {
     return {};
   }
   return {
-    borderRadius: `${borderRadius.topLeft}px ${borderRadius.topRight}px ${borderRadius.bottomRight}px ${borderRadius.bottomLeft}px`,
+    borderRadius: `${getCornerPercentages(
+      borderRadius,
+      width
+    )} / ${getCornerPercentages(borderRadius, height)}`,
```

Comment:
Are we sure we should be using `/`? Seems like it's meant for elliptical radii.

> followed optionally by "/" and one, two, three, or four <length> or <percentage> values. This is used to set an additional radius, so you can have elliptical corners.

https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius

Reply:
Hmm, I'm quite certain that for converting one `px` value to `%` we would need to use that. If the user sets one value for the corner, e.g. `20px`, and the width of the element is `400px` and the height `200px` then for converting it into `%` we would need two values for that corner: `20%` and `10%` -- this would match exactly the 20px the user set but in a responsive way.
I'm not aware of an alternative. Were you thinking of something specific? Thoughts?

Reply:
`border-radius: 20%` could be considered a shortcode for `border-radius: 20% 20% 20% 20% / 20% 20% 20% 20%` -- the `/` doesn't have to be used for something fancy. In our case, we do need to use the `/` since we need to determine both sides of each corner to keep it responsive based on the width/height. It would work without `/` only if `width === height`.

Reply:
Ah I see, thanks for explaining. Can you add a comment that describes this?